frontend/src/charts/TableChart.tsx

```diff
@@ -173,7 +175,16 @@ export function TableChart(props: TableChartProps) {
     paddingBottom: 10,
   }
 
-  const sentenceCaseName = ['hiv', 'covid', 'copd'].some((substring) =>
+  const metricSubstrings = [
```

Comment:
maybe we should consider having two properties on the MetricConfig, one a string for at the front of a sentence and one for use in the middle of the sentence? It feels weird to do all this string manipulation just to get it to capitalize properly, but it's fine for now. Maybe when we tackle re-organizing the metric config